mmv1/products/datastream/api.yaml

```diff
@@ -369,3 +367,685 @@ objects:
+                              - !ruby/object:Api::Type::Array
+                                name: 'mysqlColumns'
+                                required: true
+                                min_size: 1
+                                description: |
+                                  MySQL columns in the schema. When unspecified as part of include/exclude objects, includes/excludes everything.
+                                item_type: !ruby/object:Api::Type::NestedObject
+                                  description: |
+                                    MySQL Column.
+                                  properties:
+                                    - !ruby/object:Api::Type::String
+                                      name: 'column'
+                                      description: |
+                                        Column name.
+                                    - !ruby/object:Api::Type::String
+                                      name: 'dataType'
+                                      description: |
+                                        The MySQL data type. Full data types list can be found here:
+                                        https://dev.mysql.com/doc/refman/8.0/en/data-types.html
+                                    - !ruby/object:Api::Type::Integer
+                                      name: 'length'
+                                      description: |
+                                        Column length.
+                                    - !ruby/object:Api::Type::String
+                                      name: 'collation'
+                                      description: |
+                                        Column collation.
+                                    - !ruby/object:Api::Type::Boolean
+                                      name: 'primaryKey'
+                                      description: |
+                                        Whether or not the column represents a primary key.
+                                    - !ruby/object:Api::Type::Boolean
+                                      name: 'nullable'
+                                      description: |
+                                        Whether or not the column can accept a null value.
+                                    - !ruby/object:Api::Type::Integer
+                                      name: 'ordinalPosition'
+                                      description: |
+                                        The ordinal position of the column in the table.
+              - !ruby/object:Api::Type::Integer
+                name: 'maxConcurrentCdcTasks'
+                send_empty_value: true
+                description: |
+                  Maximum number of concurrent CDC tasks. The number should be non negative.
+                  If not set (or set to 0), the system's default value will be used.
+          - !ruby/object:Api::Type::NestedObject
+            name: 'postgresqlSourceConfig'
```

Comment:
Can we also add oracle support in this cl?

Reply:
sure, I'll put that back in since you can manually test it.